serverless.yml

```diff
@@ -27,7 +27,6 @@ functions:
 
 plugins:
   - serverless-webpack
-  - serverless-dotenv-plugin
```

Comment:
You need to add `useDotenv: true` in order to replace the functionality of `serverless-dotenv-plugin`. See the docs I linked previously.